site/docs/uk/hosting/vps.md

```diff
@@ -9,30 +9,130 @@ next: false
 
 Віртуальний приватний сервер, відомий переважно як VPS, є віртуальною машиною, яка працює в хмарі, а користувачі мають повний контроль над її системою.
 
-У цьому посібнику ви дізнаєтеся про різні методи запуску вашого бота на VPS, підтримання його в онлайні 24/7, автоматичний запуск при завантаженні вашого VPS і перезапуск при збоях.
+## Оренда сервера
 
-## systemd
+> Щоб слідувати цьому посібнику, спершу вам потрібно орендувати VPS.
```

Comment:
```suggestion
> Перш ніж розпочати ознайомлення з посібником, вам потрібно мати VPS.
```

Reply:
```suggestion
> Перш ніж розпочати ознайомлення з посібником, вам потрібно орендувати VPS.
```

Reply:
I agree. It is up to @deptyped to make the final decision.